alphagov/collectd-graphite-boshrelease
# [#104107182] Graphite for collectd boshrelease

https://www.pivotaltracker.com/n/projects/1275640/stories/104107182
1st commit: cleanup everything unrelated to collectd and add graphite configuration.

# Testing
1. checkout
2. cd to the dir, `bosh create release --name <a_name_you_can_pick> --version <you_can_also_define_this>`
3. `bosh upload release`
4. edit CF manifest. Add collectd into releases, add graphite.server property into properties and add collectd job to some server, e.g. router
5. deploy. You should start getting collectd metrics in graphite afterwards

# Reviewing
not @mtekel
